prql-compiler/src/lib.rs

```diff
@@ -11,9 +11,14 @@ pub use anyhow::Result;
 #[cfg(feature = "cli")]
 pub use cli::Cli;
 pub use error::{format_error, FormattedError, SourceLocation};
+use once_cell::sync::Lazy;
```

Comment:
This creates a static variable at runtime that is initialized exactly once. This is because we parse the version from an env string, and we don't want to do that for every request. Maybe you are familiar with `lazy_static`, this does basically the same thing.

Also, once_cell is already a transitive dependency to most crates, including this, so adding it is free :)